source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -313,6 +314,9 @@ def __init__(self, *args, **kwargs):
         self.parsers['get'].add_argument("disable_geojson", type=boolean, default=False,
                             description="remove geojson from the response")
 
+        self.parsers['get'].add_argument("from", type=coord_format,
+                                         description="Coorddinates longitude:latitude from where you want to search")
```

Comment:
longitude **;** latitude no ?
and typo on coordinate

in this wouldn't it be nice to add the fact that the coordinate is used to prioritize the objects around this coordinate ?